pkg/apis/serving/v1alpha1/revision_types.go

```diff
@@ -337,3 +342,94 @@ func (rs *RevisionStatus) GetConditions() duckv1alpha1.Conditions {
 func (rs *RevisionStatus) SetConditions(conditions duckv1alpha1.Conditions) {
 	rs.Conditions = conditions
 }
+
+const (
+	AnnotationParseErrorTypeMissing = "Empty"
```

Comment:
Is this name intentional? Missing != Empty

Reply:
Nope, good find. I'll change the string to "Missing".